src/app/core.ts

```diff
@@ -70,13 +70,26 @@ export const initDragDrop = (): Promise<boolean> => {
  * will be associated with this ID. It is recommended to use a unique ID to keep transfer
  * information private from other websites.
  *
+ * @param userSession whether or not to use multiple user sessions (defaults to false). This is
+ * useful to handle multiple user sessions in the same device.
```

Comment:
Think it would be clearer to just say when enabled, the SDK will iterate over a port range and generate a session id to determine the running instance of the desktop app for the current user. This is needed when multiple users may be logged into the same machine simultaneously, for example on a Windows Server.

Reply:
Done!